File: docs/execution/test-forxy.py
import requests
import time

with open("forxy-payload.json", 'r') as file:
    payload = file.read()

url = "http://localhost:1480/http/fork"

start = time.time()

response = requests.get(url, data=payload)

print(f"Status code: {response.status_code} Response body: {response.text}")

end = time.time()
print("Execution time: " + str(end-start))

File: docs/execution/test-forxy.php
<?php

$payload = file_get_contents("forxy-payload.json");

$url = "http://localhost:8080/http/fork";

$ch = curl_init($url);

curl_setopt($ch, CURLOPT_HTTPGET, true);
curl_setopt($ch, CURLOPT_POSTFIELDS, $payload); 
curl_setopt($ch, CURLOPT_HTTPHEADER, array(
    'Content-Type: application/json'
));

$start = microtime(true);

$response = curl_exec($ch);
$http_status = curl_getinfo($ch, CURLINFO_HTTP_CODE);

echo "Status code: $http_status\n Response body: $response\n";

$end = microtime(true);
curl_close($ch);

echo ("Execution time: " . ($end-$start));